### DISCUSSION TITLE
Any tooltip that you can recommend me that is working on latest nuxt?

### DISCUSSION THREAD

**Discussion by iamshulz** [nuxt/nuxt#30258] [Questions]
I am trying to implement my own tooltip but I decided that it's quite a hassle if I make it from the scratch.

So I am looking for suggestions. Please don't recommend `nuxt/ui` as it has issues with `nuxt.config.ts` on tailwind css.

**Comment by TheAlexLichter** [comment]
Nuxt UI is a valid option if you need a component library.

If you *only* need a tooltip component, I'd go with https://vue-tippy.netlify.app/

**Comment by renardsas** [comment]
If you don't need a component library : floating ui 
[https://floating-ui.com/docs/vue](https://floating-ui.com/docs/vue)
